gobblin-test-harness/src/test/java/org/apache/gobblin/TaskErrorIntegrationTest.java

```diff
@@ -114,24 +113,24 @@ public void testErrorDuringSubmission()
 
     GobblinLocalJobLauncherUtils.invokeLocalJobLauncher(jobProperties);
 
-    Assert.assertTrue(testAppender.events.stream().anyMatch(e -> e.getRenderedMessage()
+    Assert.assertTrue(testAppender.events.stream().anyMatch(e -> e.getMessage().getFormattedMessage()
         .startsWith("Could not submit task for workunit")));
 
     logger.removeAppender(testAppender);
   }
 
   @Test
   public void testCustomizedTaskFrameworkFailureInTaskCreation() throws Exception {
-    TestAppender testAppender = new TestAppender();
-    Logger logger = LogManager.getLogger(GobblinMultiTaskAttempt.class.getName() + "-noattempt");
+    TestAppender testAppender = new TestAppender(GobblinMultiTaskAttempt.class.getName() + "-noattempt");
+    Logger logger = (Logger) LogManager.getLogger(GobblinMultiTaskAttempt.class.getName() + "-noattempt");
```

Comment:
Maybe change to 
Logger logger = (Logger) LogManager.getLogger(testAppender.getName()); ?

also in all the places above

Reply:
i tired changing the name but the test case kept failing, even though there should not be any reason to fail, but not sure what is going on, I will check on that later, for now, I will have to keep the same.